arch/xtensa/core/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ add_custom_command(OUTPUT ${CORE_ISA_DM}
 set(ZSR_H ${CMAKE_BINARY_DIR}/zephyr/include/generated/zsr.h)
 add_custom_command(OUTPUT ${ZSR_H} DEPENDS ${CORE_ISA_DM}
   COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/gen_zsr.py
+                               $<$<BOOL:${CONFIG_XTENSA_MMU}>:--mmu>
```

Comment:
We don't have anything else right now. I have changed it to parse .config, but honestly, it was overkill for just checking if MMU is enabled. I will keep it in my stash in case we need for something else though.